src/MonitoringFunctions.Windows/Incidents/Models/Impact.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace MonitoringFunctions.Incidents.Models
+{
+    /// <summary>
+    /// Describes an incident's individual impact.
+    /// </summary>
+    [DataContract]
+    internal struct Impact : IEquatable<Impact>
+    {
+        /// <summary>
+        /// <see cref="string"/> describing the impact.
+        /// </summary>
+        [DataMember(Name = "impact-description")]
```

Comment:
@bozturkMSFT thanks for your comment.
newton soft & other serialization libraries support datamemberattribute.  newtonsoft attributes not as widespread in adoption.  We discussed enabling a yaml serializer.  

I'll change the attributes if you feel strongly enough about it. Otherwise, I'd like to leave our options